village/src/components/SmurfForm.js

```diff
@@ -11,14 +12,23 @@ class SmurfForm extends Component {
   }
 
   addSmurf = event => {
-    event.preventDefault();
-    // add code to create the smurf using the api
-
-    this.setState({
-      name: '',
-      age: '',
-      height: ''
-    });
+    console.log(event)
+    const newSmurf = {name:this.state.name, age: this.state.age, height: this.state.height}
+    console.log(newSmurf)
+    axios.post(`http://localhost:3333/smurfs`, newSmurf)
+      .then (smurf => {
+        this.setState({ smurfs:
```

Comment:
you're setting state locally on form, but to update your smurfs list, you would need to update it on App, using a method you passed down on props